internal/federation/federatingdb/create.go

```diff
@@ -116,134 +118,194 @@ func (f *federatingDB) activityBlock(ctx context.Context, asType vocab.Type, rec
 	CREATE HANDLERS
 */
 
-func (f *federatingDB) activityCreate(ctx context.Context, asType vocab.Type, receivingAccount *gtsmodel.Account, requestingAccount *gtsmodel.Account) error {
+// activityCreate handles asType Create by checking
+// the Object entries of the Create and calling other
+// handlers as appropriate.
+func (f *federatingDB) activityCreate(
+	ctx context.Context,
+	asType vocab.Type,
+	receivingAccount *gtsmodel.Account,
+	requestingAccount *gtsmodel.Account,
+) error {
 	create, ok := asType.(vocab.ActivityStreamsCreate)
 	if !ok {
-		return errors.New("activityCreate: could not convert type to create")
+		return gtserror.Newf("could not convert asType %T to ActivityStreamsCreate", asType)
 	}
 
-	// create should have an object
-	object := create.GetActivityStreamsObject()
-	if object == nil {
-		return errors.New("Create had no Object")
+	// Create must have an Object.
+	objectProp := create.GetActivityStreamsObject()
+	if objectProp == nil {
+		return gtserror.New("Create had no Object")
```

Comment:
```suggestion
		return gtserror.New("create had no Object")
```

Reply:
holy shit github does suggestions now?! i loved this at my old company that used bitbucket

Reply:
Oh yeah. They added that a while ago but it's really flown under the radar. When you start a comment, the first icon in the bar is a thing that looks like a piece of paper with a plus and minus sign in it. If you click it, it adds this:

````
```suggestion
		return gtserror.New("Create had no Object")
```
````

Then you can edit that and it shows up as a diff/suggestion you can accept.